src/librbml/leb128.rs

```diff
@@ -56,51 +58,53 @@ pub fn read_unsigned_leb128(data: &[u8], start_position: usize) -> (u64, usize)
 }
 
 
-pub fn write_signed_leb128(out: &mut Vec<u8>, start_position: usize, mut value: i64) -> usize {
-    let mut position = start_position;
+pub fn write_signed_leb128(out: &mut Vec<u8>, start_position: usize, value: i128) -> usize {
+    write_unsigned_leb128(out, start_position, value as u128)
```

Comment:
Won't this change how signed numbers are serialized? For instance `-1` was written as `7f`, but now would become `ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff 03`.